fontforge/python.c

```diff
@@ -293,15 +293,20 @@ static PyFF_Layer *LayerFromLayer(Layer *,PyFF_Layer *);
 /* ************************************************************************** */
 /* Find python objects corresponding to non-python structures */
 /* ************************************************************************** */
-static PyFF_Font* PyFF_FontForFV( FontViewBase *fv ) {
+PyObject* PyFF_FontForFV( FontViewBase *fv ) {
     if ( fv==NULL )
 	return NULL;
+    if ( fv->python_fv_object==NULL ) {
+        fv->python_fv_object = PyFF_FontType.tp_alloc(&PyFF_FontType,0);
+        ((PyFF_Font *) (fv->python_fv_object))->fv = fv;
+    }
+    Py_INCREF( (PyObject *) (fv->python_fv_object) );
     return fv->python_fv_object;
 }
 static PyFF_Font* PyFF_FontForSF( SplineFont *sf ) {
     if ( sf==NULL )
 	return NULL;
-    return PyFF_FontForFV( sf->fv );
+    return (PyFF_Font*)PyFF_FontForFV( sf->fv );
```

Comment:
@ctrlcctrlv You're now decrementing in the methods you've *changed*, but you haven't taken into account the other already present callers of `PyFF_FontForFV()` -- for example this one.

Reply:
Oh yeah :) OK maybe we do need a `PyFF_FontForFV_I`